src/fuzz/protobufs/utils/spanner_emulator_ddl_statement_proto_to_string.h

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "src/fuzz/protobufs/create_table.pb.h"
+#include "src/fuzz/protobufs/spanner_ddl.pb.h"
+
+#include <google/protobuf/repeated_field.h>
+
+#include <string>
+#include "absl/strings/substitute.h"
+#include "absl/strings/str_cat.h"
+
+using spanner_ddl::SpannerDDLStatement;
+using spanner_ddl::CreateTable;
+using spanner_ddl::Column;
+using spanner_ddl::ColumnDataInfo;
+
+// proto to string methods
+std::string toString(SpannerDDLStatement statement);
+std::string toString(CreateTable createTable);
+std::string toString(const google::protobuf::RepeatedPtrField<Column> columns);
```

Comment:
Pass by const ref?

Reply:
In particular, it should work to do "const Column& column" here.

Reply:
Rather, no--use "const google::protobuf::RepeatedPtrField<Column>&" or similar.